### PR TITLE
Border radius

### PR DESCRIPTION
**Pull request by TigranH13**

### CODE REVIEW

File: lib/presentation/theme/border_radius/widgets/border_radius_scheme.dart [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    final List<Map<String, double>> borderRadius = [
+      {'Br 1': 2},
+      {'Br 2': 4},
+      {'Br 3': 8},
+      {'Br 4': 14},
+      {'Br 5': 16},
+      {'Br 6': 20},
+      {'Br 7': 32},
+      {'All': 500},
+    ];
```
Comment (narekv7) [inline lib/presentation/theme/border_radius/widgets/border_radius_scheme.dart:22]: now we can use BR from package

**Review by narekv7 — CHANGES_REQUESTED** [review]

File: lib/presentation/example_screens/screens/checkout_screen.dart [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            color: t.colors.neutralHighContent,
+          ),
+        ),
+        const SizedBox(height: 20),
+        Container(
+          padding: EdgeInsets.all(t.spacing.x4),
+          decoration: BoxDecoration(
+            color: t.colors.surface,
+            borderRadius: BorderRadius.circular(16),
+          ),
```
[the remark this answers is not in the document]
Comment (TigranH13) [reply lib/presentation/example_screens/screens/checkout_screen.dart:33]: its your work

File: lib/presentation/example_screens/screens/love_screen.dart [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                  backColor: context.mdsTheme.colors.neutralLowContainer,
+                  children: [
+                    Text(
+                      '1.Physical toch.',
+                      style: textTheme.bodyMRegular,
+                    ),
+                    Checkbox(
+                      value: false,
+                      onChanged: (value) {},
+                    )
```
[the remark this answers is not in the document]
Comment (TigranH13) [reply lib/presentation/example_screens/screens/love_screen.dart:72]: its your work

File: lib/presentation/example_screens/screens/love_screen.dart [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                const SizedBox(height: 16),
+                ExampleItem(
+                  padding: const EdgeInsets.all(0),
+                  backColor: context.mdsTheme.colors.neutralLowContainer,
+                  children: [
+                    Text(
+                      '2. Receiving gifts.',
+                      style: textTheme.bodyMRegular,
+                    ),
+                    Checkbox(
```
[the remark this answers is not in the document]
Comment (TigranH13) [reply lib/presentation/example_screens/screens/love_screen.dart:84]: its your work

File: lib/presentation/theme/border_radius/widgets/border_radius_scheme.dart (hunk 1 above)
Comment (TigranH13) [reply lib/presentation/theme/border_radius/widgets/border_radius_scheme.dart:22]: it for make easier my work with ui (all wigets draw with one for loop)

File: lib/presentation/example_screens/screens/checkout_screen.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (TigranH13) [reply lib/presentation/example_screens/screens/checkout_screen.dart:33]: you merge that branch to main without review and even it wasnt in dev

File: lib/presentation/theme/border_radius/widgets/border_radius_scheme.dart (hunk 1 above)
Comment (narekv7) [reply lib/presentation/theme/border_radius/widgets/border_radius_scheme.dart:22]: bodrer radius -es should depend on theme , so remove hardcoded values

File: lib/presentation/example_screens/screens/love_screen.dart (hunk 3 above)
[the remark this answers is not in the document]
Comment (narekv7) [reply lib/presentation/example_screens/screens/love_screen.dart:72]: its a mistake , replace

**Review by narekv7 — APPROVED** [review]

**Pull request merged**